internal/driver/glfw/driver_desktop.go

```diff
@@ -40,7 +40,7 @@ func (d *gLDriver) SetSystemTrayMenu(m *fyne.Menu) {
 			} else if fyne.CurrentApp().Icon() != nil {
 				d.SetSystemTrayIcon(fyne.CurrentApp().Icon())
 			} else {
-				d.SetSystemTrayIcon(theme.FyneLogo())
+				d.SetSystemTrayIcon(theme.WarningIcon())
```

Comment:
I suppose it is worth discussing what to do about systray. Should we use a warning icon, a question icon or just not set one at all?

Reply:
We have to set something surely? Without it there will be no display in the tray.

Reply:
I guess it depends on how the system tray handles no icons. At least for app icons, a "no icon" icon is usually displayed if you don't provide one. Perhaps the same thing happens with system tray?

Reply:
Sadly it doesn't. At least not on some Os. Those just display a blank space that can be tapped. And i think I've seen some make it quite narrow as well (like it's "" instead of a missing icon).

Reply:
I have switched to use the `BrokenImageIcon` from fynedesk.